Q/Qt/build_tarballs.jl

```diff
@@ -3,18 +3,21 @@
 using BinaryBuilder
 
 name = "Qt"
-version = v"5.15.0"
+version = v"5.15.1"
 
 # Collection of sources required to build qt5
 sources = [
     ArchiveSource("https://download.qt.io/official_releases/qt/5.15/$version/single/qt-everywhere-src-$version.tar.xz",
```

Comment:
You can fully parametrise the URL with
```suggestion
    ArchiveSource("https://download.qt.io/official_releases/qt/$(version.major)$(version.minor)/$version/single/qt-everywhere-src-$version.tar.xz",
```

Reply:
Done.